specification/batch/data-plane/readme.md

```diff
@@ -274,6 +274,24 @@ batch:
   - tag: package-2017-05.5.0
 ```
 
+### Tag: package-2017-09.6.0 and go

```

Comment:
These are missing the month, is that intentional?  I think we should keep it so it's unifor with 2017-05-01.5.0 which is already published.

Reply:
Fixed in 4d86a78